packages/fxa-auth-server/tsconfig.json

```diff
@@ -8,12 +8,19 @@
     "checkJs": false,
     "outDir": "./dist",
     "types": ["accept-language", "mocha", "mozlog", "node", "fxa-geodb"],
-    "lib": ["ESNext"]
+    "lib": ["ESNext"],
+    // We should remove this, but for whatever reason, esbuild was not complaining
+    // about these explicit any
+    "noImplicitAny": false
   },
   "include": [
-    "bin/*",
-    "scripts/**/*.ts",
-    "scripts/delete-account.js",
-    "lib/senders/emails/templates/*/includes.ts"
+    "bin/**/*.js",
+    "bin/**/*.ts",
+    "lib/**/*.ts",
+    "lib/**/*.js",
+    "scripts/*.ts",
+    "scripts/*.js",
+    "test/**/*.ts",
+    "test/**/*.js"
```

Comment:
Confirmed no weird .jsx or .tsx files.

```sh
git ls-files packages/fxa-auth-server/(bin|lib|scripts|test)/ | grep -Eo "\.[a-z]+$" | sort | uniq -c | sort -rn
 343 .js
 197 .ts
 139 .json
  85 .txt
  77 .mjml
  76 .ftl
   8 .scss
 ...
```

Reply:
Thanks for checking. Server so, hopefully not!